### DISCUSSION TITLE
Как часто обновляются месячные цены?

### DISCUSSION THREAD

**Discussion by DizzyJump** [mbk-dev/okama#7] [Q&A] [answered]
Добрый день, подскажите пожалуйста как часто обновляются месячные цены активов?
Например сегодня первое февраля, я запрашиваю список активов с августа по январь:
```
ls = ['FXGD.MOEX','VTBX.MOEX','VTBE.MOEX','SBGB.MOEX','RUSB.MOEX','FXTB.MOEX','FXDE.MOEX','FXCN.MOEX','FXUS.MOEX']
asl = ok.AssetList(ls, curr='USD', first_date='2020-08-01', last_date='2021-01-01')
```
и получаю ответ:
```
symbols              [FXGD.MOEX, VTBX.MOEX, VTBE.MOEX, SBGB.MOEX, R...
currency                                                           USD
first date                                                     2020-08
last_date                                                      2020-12
period length (Y)                                                  0.3
inflation                                                     USD.INFL
```

т.е. последняя цена за декабрь... а когда примерно появятся цены за январь?

**Comment by chilango74** [comment, marked as answer]
@DizzyJump , большинство тикеров обновляются ежедневно.
Но ... Сроки данных в `AssetList` ограничены инфляцией. Сейчас есть официальные показатели по инфляции только за декабрь 2020. Январские данные будут опубликованы примерно в 20х числа февраля. 
Но вы можете использовать флаг `inflation=False` в сигнатуре AssetList, тогда инфляция не будет учитываться.

Вы всегда можете проверить, какой именно символ являются причиной ограничения сроков данных через `.assets_first_dates` и `.assets_last_dates`:
![image](https://user-images.githubusercontent.com/29829921/106564358-2e0e5100-653e-11eb-8e7f-2d9119cefbef.png)

П.С. Есть отдельный [русскоязычный форум](https://community.okama.io/c/python-okama), где ведется обсуждение okama.

**Reply by DizzyJump** [reply]
Спасибо!
